src/routes/therapist.ts

```diff
@@ -8,7 +8,7 @@ const router = express.Router();
 router.get('/test-auth', checkAuth(RolesForSelect.user), (req, res) => {
   res.json('hi auth');
 });
-router.get('/:id', findTherapistById);
 router.get('/', getAllTherapists);
-
+router.get('/:id', findTherapistById);
+router.patch('/:id', checkAuth(RolesForSelect.therapist), updateTherapistProfile);

```

Comment:
```suggestion
router.patch('/', checkAuth(RolesForSelect.therapist), updateTherapistProfile);

```
why we do need `:id` params as we're not depending on it

Reply:
Yes right, and DONE.